src/data/pnat/basic.lean

```diff
@@ -3,39 +3,29 @@ Copyright (c) 2017 Microsoft Corporation. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro, Neil Strickland
 -/
+import data.pnat.defs
 import data.nat.order
 import algebra.order.positive.ring
 
 /-!
 # The positive natural numbers
 
-This file defines the type `ℕ+` or `pnat`, the subtype of natural numbers that are positive.
+This file develops the type `ℕ+` or `pnat`, the subtype of natural numbers that are positive.
+It is defined in `data.pnat.defs`, but most of the development is deferred to here so
+that file can have very few imports.
```

Comment:
```suggestion
that `data.pnat.defs` can have very few imports.
```
I had to read the sentence as it stood twice before I could make sense of it.

Reply:
I think it was missing a second `that`, but your phrasing is better anyway.